Bottom: a1379b82e2fe51b5c630e7b7a03b7cbe8c652618
Top:    6f2da6758409d9ca509897a7ee37ed396d10bae6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-24 01:16:33 +0100

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index 92c8b35..765aeec 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2747,7 +2747,12 @@ IsPolygonInPolygon (PolygonTypePtr P1, PolygonTypePtr P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
